src/chia_dialect.rs

```diff
@@ -12,12 +13,12 @@ use crate::more_ops::{
 use crate::reduction::Response;
 
 pub struct ChiaDialect {
-    strict: bool,
+    flags: u32,
```

Comment:
Any reason not to simply take a list of booleans? I know `flags` is quite traditional, but let me ask you: when have you seen an API that takes `flags` as a parameter and been pleased about it?

Reply:
generally I find flags better in all respects.

1. They take less space
2. The enforce names to be used (rather than 'true' and 'false')
3. They are easy to pass on to the next function

e.g.

```
f(true, true, false);
```

vs.

```
f(COND_CANON_INTS | NO_NEG_DIV);
```

Granted, in this specific place, rust will enforce names to be used to initialize the struct, but if used as parameters to any function, I think a series of bools would be inferior because of (1) and (3).

These flags are not type safe right now, but I expect that to be a small change to make.

Reply:
Maybe consider using [bitflags](https://docs.rs/flags/latest/bitflags/) or something to get typed flags. (I've not used this library so can't vouch for its goodness or badness.)

Reply:
yeah, I've only had a brief look at it and wasn't very impressed so far.